data-structure.py

```diff
@@ -2,22 +2,9 @@ def read_file(text_file):
   if ".txt" in text_file:
     recipe_list = open(text_file)
     opened_recipe_list = (recipe_list.read())
-    return opened_recipe_list
+    print opened_recipe_list

```

Comment:
What is the reason for printing instead of returning file data here?

Reply:
Because of this good reason I did it.